tree-sitter-darklang/grammar.js

```diff
@@ -28,6 +28,7 @@ module.exports = grammar({
   name: "darklang",
 
   externals: $ => [$.indent, $.dedent],
+  extras: $ => [/[\s\uFEFF\u2060\u200B\u00A0]/, $._inline_comment],
```

Comment:
What's this?
```suggestion
  extras: $ => [
    /[\s\uFEFF\u2060\u200B\u00A0]/, // some commentary - not sure what this is
    $._inline_comment
  ],
```

Reply:
I'll add the commentary.

The default value of extras includes whitespace; when adding comments to it, I believe we should handle whitespace explicitly. Maybe we can just have it as `[/\s/, $._inline_comment]` the others represent unicodes for zero-width spaces and non-breaking spaces, many languages I checked includes them and some don't, should we keep them?

Reply:
I'm not sure - you'd probably be the better judge on this question. :)

Reply:
The default value in tree-sitter is `[/\s/]`, so I'll just keep it simple. if we notice things breaking we can add those back